core/app/models/spree/order.rb

```diff
@@ -369,8 +369,8 @@ def paid?
       payment_state == 'paid' || payment_state == 'credit_owed'
     end
 
-    def available_payment_methods
-      @available_payment_methods ||= collect_payment_methods
+    def available_payment_methods(store = nil)
```

Comment:
I still want to keep the old interface of this method for backward compatibility and avoid some unexpected things.